### PR TITLE
Mouse UI 

### PR DESCRIPTION
**Pull request by hkaur008**
Things added so far:-
- Add a mouse interactive UI background

Reasons to add:-
- To viewers helps to gain more interest in the portfolio.

## Please, go through these steps before you submit a PR.
- [X] You've explained what the Pull Request is adding.
- [X] You've explained why this is important.
Fixes issue #15

### CODE REVIEW

File: _sass/all.scss [hunk 1]
```diff
@@ -80,6 +80,11 @@ a {
         font-size: small;
         color: var(--fg-0)
     }
+    
+    .username {
+        font-size: medium;
+        color: #404040;
```
Comment (rosekamallove) [inline _sass/all.scss:86]: Adding one of the variables instead of hard coded value would be better.

**Review by rosekamallove — CHANGES_REQUESTED** [review]
Please change hard coded color value to one of the variables in _sass/theme.scss

**Comment by rosekamallove** [comment]
This and PR #14 seem to have the same file changes and commits.

**Comment by hkaur008** [comment]
> This and PR #14 seem to have the same file changes and commits.

opps i just forget to commit and add some files

**Comment by hkaur008** [comment]
yeah implemented things out !

**Comment by awilkescom** [comment]
Hey @hkaur008 , I couldn't get this working

for some reason i get an error in chrome to load a JS file it seems,

image

Also I think for something visual, it would be worth to include a screenshot in your PR so it's clear what to look for. 

Could you see if you can debug this issue and update the PR ? thanks

**Comment by awilkescom** [comment]
Ok, the error with the JS file resolved itself, but I can't see any visual change

**Comment by hkaur008** [comment]
hey @awilkescom let me check again , i think there is were some pull request merged before which caused conflicts

**Comment by hkaur008** [comment]
hey ! @awilkescom  . you can check now

**Comment by awilkescom** [comment]
I really like it, nice effect! Thanks @hkaur008 !

**Pull request merged**